### PR TITLE
new app layout is enabled

### LINKED ISSUE #6958: App Layout: Enable Feature Flag
App Layout should be enabled by default

### PR DESCRIPTION
**Pull request by fedrunov**
## Type of change

- [x] Feature
- [ ] Bugfix
- [ ] Technical
- [ ] Other :

## Content

New App Layout should be enabled by default

## Motivation and context

closes #6958

### CODE REVIEW

**Review by bmarty — APPROVED** [review]
LGTM.

One point though: I do not see any update on the test allScreensTest().

This test has to pass before the next release with AppLayout enabled. Ideally the test can check the result of isNewAppLayoutEnabled(), to pass in both cases.

If the test is already passing, that's fine!

**Comment by fedrunov** [comment]
> One point though: I do not see any update on the test allScreensTest().

There is a separate issue for updating tests for App Layout. I'll do it as soon as I'll have all issues with new screens/changes done

https://github.com/vector-im/element-android/issues/6509

**Comment by bmarty** [comment]
> > One point though: I do not see any update on the test allScreensTest().
> 
> There is a separate issue for updating tests for App Layout. I'll do it as soon as I'll have all issues with new screens/changes done
> 
> #6509

I see, thanks. But you see the problem here, once this PR will be merged, the test will not pass, and this can block the release.
This is not blocking, but if the sanity test are not passing during the release process, we will not be able to release the feature, and it will be disabled.

**Pull request merged**
